showcase/showcase-gallery/webpack.js

```
@@ -0,0 +1,129 @@
+              },
+            },
+          ],
+        },
+        {
+          test: /\.png$/,
+          use: [{ loader: "url-loader", options: { limit: 10000 } }],
+        },
+        {
+          test: new RegExp(
+            "./node_modules/lit-element-router/utility/router-utility.js"
+          ),
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)